### DISCUSSION TITLE
I have a question about the `SecureStorageItemTypes` in the `SpeziStorage` library.

### DISCUSSION THREAD

**Discussion by chorim** [StanfordSpezi/.github#60] [Challenges and Help] [answered]
### Description

I'm looking at the internal components of the library and have a question.
I realised that the `credentials` variable on line 22, that has been contained a duplicate value.

```swift
public static let credentials: SecureStorageItemTypes = [.serverCredentials, .serverCredentials]
```

The code should look something like this?

```swift
public static let credentials: SecureStorageItemTypes = [.serverCredentials, .nonServerCredentials]
```
or
```swift
public static let credentials: SecureStorageItemTypes = [.serverCredentials]
```

### Contact Information and Additional Context

me@byeon.is

### Code of Conduct

- [X] I agree to follow this projects's Code of Conduct and Contributing Guidelines

**Comment by PSchmiedmayer** [comment, marked as answer]
@chorim Thank you for raising this issue. You are correct; that's an issue in the library. I added this fix to https://github.com/StanfordSpezi/SpeziStorage/pull/22.

Feel free to use GitHub issues as a great way to report bugs and issues that you encounter in our different models!
Feel free to use the [Show and Tell GitHub Discussions](https://github.com/orgs/StanfordSpezi/discussions/categories/show-and-tell) section to share your projects that use Spezi!
